packages/cxl-lumo-styles/scss/vaadin-details.scss

```diff
@@ -0,0 +1,34 @@
+:host([theme~="sidebar-track"]) {
+  margin: 0;
+
+  [part="summary"] {
+    padding: var(--lumo-space-l) var(--lumo-space-m);
+    border-radius: 0;
+    border-top: 2px solid #e1e1e1;
+    cursor: pointer;
+
+    display: grid;
+    grid-template-areas: "content toggle";
+    grid-template-columns: auto 20px;
+
+    [part="toggle"] {
+      grid-area: toggle;
+      width: 20px;
+      height: 19px;
+      margin-right: 0 !important;
+      margin-left: 0 !important;
+      font-size: var(--lumo-icon-size-m);
+    }
+
+    [part="summary-content"] {
+      grid-area: content;
+    }
+  }
+}
+
+:host([opened]) {
+  [part="summary"] {
+    border-bottom: 2px solid #e1e1e1;
+    box-shadow: 0px 3px 4px rgba(130, 130, 130, 0.1);
+  }
+}
```

Comment:
Struggling to understand these mechanics, need more time tomorrow.

CSS grid inside details element seems like overkill.